Update app-with-collaboration-controls.md (#7469)
MicrosoftDocs/msteams-docs: msteams-platform/samples/app-with-collaboration-controls.md

You can define settings for Collaboration controls for the business entity such as the table created in [new model-driven app](#create-a-new-model-driven-app-with-collaboration-controls-for-teams).

> [!NOTE]

> > If applying settings for the Collaboration Manager for Loans product, follow the steps as described here: https://learn.microsoft.com/en-us/industry/financial-services/collaboration-manager/configure

Reviewer: ```suggestion
> For information on how to configure Collaboration Manager for Loans product, see [Collaboration Manager for Loans](/industry/financial-services/collaboration-manager/configure).
```
